src/client/components/MovieDetailModal.jsx

```diff
@@ -9,8 +9,23 @@ export default function MovieDetailModal({ movieDetail, onClose }) {
   const { title, bannerUrl, releaseYear, genres, rate, description } =
     movieDetail;
 
+  const handleESCKeyDown = useCallback((event) => {
+    if (event.key === "Escape") {
+      onClose();
+    }
+  });
+
+  useEffect(() => {
+    document.addEventListener("keydown", handleESCKeyDown);
+    return () => document.removeEventListener("keydown", handleESCKeyDown);
+  });
```

Comment:
`useEffect`에 의존성 배열이 없는데, 추가하면 좋을 것 같습니다!

```suggestion
useEffect(() => {
  document.addEventListener("keydown", handleESCKeyDown);
  return () => document.removeEventListener("keydown", handleESCKeyDown);
}, []); // 의존성 배열 추가
```